client/signup/steps/site-or-domain/index.jsx

```diff
@@ -51,7 +51,7 @@ class SiteOrDomain extends Component {
 				label: translate( 'New site' ),
 				image: <NewSiteImage />,
 				description: translate(
-					'Choose a theme, customize, and launch your site. A free domain for one year is included with all plans.'
+					'Choose a theme, customize, and launch your site. A free domain for one year is included with all annual plans.'
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.